worlds/kh2/Locations.py

```diff
@@ -1356,5 +1356,5 @@
 location_groups: typing.Dict[str, list]
 location_groups = {
     Region_Name: [loc for loc in Region_Locs if "Event" not in loc]
-    for Region_Name, Region_Locs in KH2REGIONS.items() if Region_Locs
+    for Region_Name, Region_Locs in KH2REGIONS.items() if len(Region_Locs) > 0 and "Event" not in Region_Locs[0]
```

Comment:
```suggestion
    for Region_Name, Region_Locs in KH2REGIONS.items() if Region_Locs and "Event" not in Region_Locs[0]
```
just a tad bit faster (and i find it easier to read)